apps/expo/src/app/post/new.tsx

```diff
@@ -0,0 +1,68 @@
+import { router } from 'expo-router';
+import { useState } from 'react';
+import { KeyboardAvoidingView, Platform, Text, TextInput } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context'
+import Button from '~/components/ui/button/button';
+import { useGlobalContext } from '~/context/global-context';
+import { api } from '~/utils/trpc/api';
+
+const inputStyles = {
+  borderWidth: 1,
+  borderColor: '#FFFFFF',
+  borderRadius: 10,
+  padding: 10,
+  margin: 5,
+  color: 'white',
+  backgroundColor: '#1C1B1B',
+  opacity: 0.9,
+};
+
+const NewPost = () => {
+  const { userData } = useGlobalContext();
+  const [title, setTitle] = useState('');
```

Comment:
Executive decision we already dropped title, so hope you can accommodate for this.

Now that we only have 1 field of input which is the content.

Will it be considered something like twitter did

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/134f5457-c7c3-4e24-bdad-ad6d14a71713)

we will have an avatar and an input field like this. Of course, we don't have the bunch of icons underneath but this could be a reference.

let me know what you think